### ISSUE TITLE
my cat cannot comment in Web-Environment-Integrity

### ISSUE THREAD

**Issue by MishaProductions** [RupertBenWiser/benwiser.com#29]
Not really related to this exact repo, but my cat is complaining that "its tuesday and the comments are still not open."

**Comment by MishaProductions** [comment]
![PXL_20230725_132240789](https://github.com/RupertBenWiser/benwiser.com/assets/106913236/5a428d30-db93-4a39-b29c-e8b92f3768cb)

**Comment by lunarythia** [comment]
Unfortunately, W.E.I. is meant to make sure only humans can get in. Looks like your cat is out of luck. Cute cat though.

**Comment by ghost** [comment]
Fuck Google. Shutting down issues and everything. As soon as the repository gets unlocked, spam in it so that the issues are unmanageable. 

This proposal won't eliminate the open web because there is not an open web. But at least don't let it get worse.

**Comment by lunarythia** [comment]
Web Environment Integrity (W.E.I.) more like Web Excessive Idiocy.

**Comment by lunarythia** [comment]
> Fuck Google. Shutting down issues and everything. As soon as the repository gets unlocked, spam in it so that the issues are unmanageable.
> 
> This proposal won't eliminate the open web because there is not an open web. But at least don't let it get worse.

Yes, make sure that “legitimate” issues, whatever the damn they may be, are lost to the storm. They deserve it.
